lib/cli/consolecommand.cpp

```diff
@@ -382,10 +385,17 @@ int ConsoleCommand::RunScriptConsole(ScriptFrame& scriptFrame, const String& add
 			line = commandOnce;
 
 		if (!line.empty() && line[0] == '$') {
-			if (line == "$continue")
+			if (line == "$continue" || line == "$quit" || line == "$exit")
 				break;
+			else if (line == "$help")
+				std::cout << "Welcome to the Icinga 2 console/script debugger.\n"
```

Comment:
Please name it `debug console`, similar to the docs. Some users don't understand its purpose unless explicitly stated.